### PR TITLE
New version: LightBurnSoftware.LightBurn version 1.6.0

### PR DESCRIPTION
**Pull request by J3n50m4t**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155297)

### CODE REVIEW

**Comment by J3n50m4t** [comment]
@microsoft-github-policy-service agree

**Comment by J3n50m4t** [comment]
Unsure what to do. 
/SUPPRESSMSGBOXES in the verysilent? 
Install works on my machine with with local manifest without interaction (with progress) - exactly as 1.4.0 was.

File: manifests/l/LightBurnSoftware/LightBurn/1.6.0/LightBurnSoftware.LightBurn.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- tga
+- tif
+ReleaseDate: 2024-05-01
+AppsAndFeaturesEntries:
+- DisplayName: LightBurn version 1.6.00
+  ProductCode: LightBurn_is1
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/LightBurnSoftware/deployment/releases/download/1.6.00/LightBurn-v1.6.00.exe
+  InstallerSha256: 27FC656AE07394CBD5E5217C04BE516AB7F4646F052FBE14C87D9B6A595CE4B1
```
Comment (stephengillie) [inline manifests/l/LightBurnSoftware/LightBurn/1.6.0/LightBurnSoftware.LightBurn.installer.yaml:44]:
```suggestion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @J3n50m4t,

Our `x86` validations are performed on 32-bit OSes. This application appears to be 64-bit only.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**